### DISCUSSION TITLE
Accordion to change the icon in the menu and move to the right

### DISCUSSION THREAD

**Discussion by cahitberkay** [fomantic/Fomantic-UI#1347] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
Hi! 

I wanted to make changes to the Accordion menu, but I can do it in React. Can it be used outside of React? For example I want to change the icon

The example I want to do;
![image](https://user-images.githubusercontent.com/24186944/75375615-7f9a6e80-58df-11ea-95b6-e9c9117bc3b3.png)

**Comment by lubber-de** [comment]
Sure. See https://jsfiddle.net/dk85Lxse/

**Comment by cahitberkay** [comment]
This way I can do it too. justify-content; with space-between ..

But I wanted to learn if there is anything ready for the plus and minus to change constantly. Without writing extra JS code ..

**Comment by lubber-de** [comment, marked as answer]
You can/have to override the classes for `.ui.accordion > .title > .icon` and `.ui.accordion > .active.title >.icon` without using javascript
There is currently no existing option  which would do this out of the box

**Comment by cahitberkay** [comment]
Okey, thank you :)
